Docs 672 small corrections

rebekah-lawrence opened the pull request:
Updated from [request from Yuce on Slack](https://hazelcast.slack.com/archives/C035HQET5/p1697459969855129?thread_ts=1697286884.502209&cid=C035HQET5)

rebekah-lawrence committed:
DOCS-672_small_corrections

Removed sql from clc shell command

Re-ordered to swap steps 4 and 5

rebekah-lawrence committed:
DOCS-672_list_order

Rearranged the order of the list introducing the tutorial to match the new flow

yuce commented on docs/modules/ROOT/pages/sql-on-viridian.adoc:
```diff
@@ -197,61 +197,7 @@ ON companies.ticker = trades.ticker;
-
-
-== Step 5: Stream to Stream Joins
+== Step 4: Stream to Stream Joins
```
@rebekah-lawrence Thanks for this PR!

I think I've explained "step 4" in a wrong way.
What I meant was, it'd better to move "Step 4" of "Step 1. Set up Streaming Data" after "Step 5" of "Step 1. Set up Streaming Data" section.
Since the code in "Step 3" of "Step 1. Set up Streaming Data" has the `CREATE OR REPLACE VIEW` it will complete in a few seconds, and the user shouldn't press Ctrl+C when that runs.

So, I think the ideal order in "Step 1. Set up Streaming Data" would be:
3. Use an SQL view to generate streaming data.
4. Verify that the data is streaming by issuing a query.
5. If you’re using the CLC SQL shell, press CTRL-C to return to the prompt.

rebekah-lawrence replied:
I was actually going to double-check whether that was what you meant when I looked at it yesterday, but got side-tracked and completely forgot today!

Ignore this one, I'll try again. Apologies.

rebekah-lawrence committed:
DOCS-672_correcting_my_misunderstanding

Should now show the actual updates and the undoing of the re-ordering of the Step 4/5 headings
